app/src/main/res/values-night/colors.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="colorPrimary">#212121</color>
-    <color name="colorAccent">#eeeeee</color>
+    <color name="colorAccent">#899ab4</color>
```

Comment:
use                 messageHolder.contentView.setLinkTextColor(ContextCompat.getColor(context, R.color.color_here));

in recyclerMessageAdapter!